grapple/urls.py

```diff
@@ -26,8 +25,8 @@ def graphiql(request):
 
 # Traditional URL routing
 urlpatterns = [
-    url(r"^graphql", csrf_exempt(GraphQLView.as_view()), name="grapple_graphql")
+    re_path(r"^graphql", csrf_exempt(GraphQLView.as_view()), name="grapple_graphql")
 ]
 
 if grapple_settings.EXPOSE_GRAPHIQL:
-    urlpatterns.append(url(r"^graphiql", graphiql, name="grapple_graphiql"))
+    urlpatterns.append(re_path(r"^graphiql", graphiql, name="grapple_graphiql"))
```

Comment:
it should not need `re_path`, `path("graphiql")` and `path("graphql")` should do it

Reply:
Updated!